docling/models/genos_dots_ocr_layout_model.py

```
@@ -946,6 +956,111 @@ def _process_page(self, conv_res: ConversionResult, page: Page) -> Page:
+            return self._tableformer_model
+        self._tableformer_model = None
+        try:
+            from docling.models.table_structure_model import TableStructureModel
+
+            self._tableformer_model = TableStructureModel(
+                enabled=True,
+                artifacts_path=getattr(self.pipeline_options, "artifacts_path", None),
+                options=self.pipeline_options.table_structure_options,
+                accelerator_options=self.pipeline_options.accelerator_options,
+            )
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)